components/suggestion/suggestion_box.jsx

```diff
@@ -656,7 +656,7 @@ export default class SuggestionBox extends React.PureComponent {
             if (handled) {
                 if (this.state.suggestionBoxAlgn.pixelsToMoveX === undefined &&
                     this.state.suggestionBoxAlgn.pixelsToMoveY === undefined &&
-                    provider.triggerCharacter && ['@', ':', '~'].indexOf(provider.triggerCharacter) !== -1) {
+                    ['@', ':', '~', '/'].includes(provider.triggerCharacter)) {
```

Comment:
hi @hahmadia , the includes function has no support for IE and I think babel doesn't transpile it either. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#browser_compatibility

Reply:
Hey @pablovelezvidal !

Thanks for the point out but I think we should be okay. Let me know otherwise. See https://forum.mattermost.org/t/mattermost-is-dropping-support-for-internet-explorer-ie11-in-v5-16/7575

Reply:
ok, perfect, I was not entirely sure of the MM support to IE, then we are fine by using Array includes prototype.

Reply:
One of the best things about working on Mattermost is that it doesn't support IE any more 😄